linkerd/app/inbound/src/policy/discover.rs

```diff
@@ -109,6 +109,21 @@ fn to_policy(proto: api::Server) -> Result<ServerPolicy> {
         _ => return Err("proxy protocol missing".into()),
     };
 
+    let loopback = Authorization {
+        name: "default:localhost".into(),
+        authentication: Authentication::Unauthenticated,
+        networks: vec![
+            Network {
+                net: "127.0.0.1".parse::<IpAddr>().unwrap().into(),
```

Comment:
nit/TIOLI: maybe a bit nicer to just write
```suggestion
                net: IpAddr::from([127, 0, 0, 1]).into(),
```
rather than trying to parse a hardcoded string? not that parsing the string will ever fail, but it's nicer to not parse things that don't need to be parsed IMO...